requirements.txt

```diff
@@ -73,12 +73,10 @@ cryptography==3.3.2
 dnspython==1.16.0
     # via eventlet
 docopt==0.6.2
-    # via
-    #   -r requirements.in
-    #   notifications-python-client
+    # via notifications-python-client
 docutils==0.15.2
     # via awscli
-eventlet==0.30.2 # pyup: ignore
+eventlet==0.30.2
```

Comment:
dunno why this line has changed 🤔

Reply:
I think we should put it back to stop PyUp trying to upgrade it.

Reply:
hmmm i'm not sure, i don't think that's needed - we still have the line in requirements.in (https://github.com/alphagov/notifications-api/blob/master/requirements.in#L13) . we don't have any of the other pyup lines reflected in this file (cryptography, marshmallow, etc are all pinned with `# pyup:< 1.2.3` comments in reqs.in)

Reply:
Maybe it's this: https://github.com/alphagov/notifications-admin/commit/fab44e016a2d65ff04675a139c5a8386dfd8a1d2

Reply:
Aha, yeah, that would explain why you're now getting the safety warning!

Reply:
ahhh good ifnd. i'll re-add that back in.